schema.js

```diff
@@ -19,7 +19,10 @@ module.exports = Joi.object().keys({
       api_key: Joi.string(),
       importVenues: Joi.boolean().default(false).truthy('yes').falsy('no').insensitive(true),
       importPostalcodes: Joi.boolean().default(false).truthy('yes').falsy('no').insensitive(true),
-      missingFilesAreFatal: Joi.boolean().default(false).truthy('yes').falsy('no').insensitive(true)
+      missingFilesAreFatal: Joi.boolean().default(false).truthy('yes').falsy('no').insensitive(true),
+      sqlite_files: Joi.array().items(Joi.object().keys({
+        filename: Joi.string()
+      }).requiredKeys('filename').unknown(true)),
```

Comment:
What would it take to remove the need for the new `sqlite_files` `pelias.json` property entirely? Previously, the importer was able to determine what combination of admin, postalcode, and venue bundles to download. Can we do the same with the SQLite downloads?

Reply:
Good idea, that's pretty easy to implement and would be super convenient.

We'd need to talk about how the paths would work, currently `datapath` refers to the directory which has `data` and `meta` as it's children.

I have `sqlite_files.filename` set up to point to an absolute path. In order to get the downloads working, we would probably need to change that to only list the filename, not the path.

Then the question is, which path to we download to and load from? we could use `datapath` or we could introduce a new path param?

I'm currently just downloading my databases in to the `datapath` and it looks like this:

```
$ ls -1
data
meta
whosonfirst-data-latest.db
whosonfirst-data-postalcode-us-latest.db
```

It's probably fine to use `datapath` for most use-cases, we just need to make users aware that the files can be quite large.

```
15G whosonfirst-data-latest.db
1.4G whosonfirst-data-postalcode-us-latest.db
```

Reply:
Yeah, putting the DBs into `datapath` is perfect. Eventually, if/when we switch to importing straight from sqlite, rather than generating the files from the sqlite DB, everything would already be in the right place and the `data` and `meta` directories would disappear. As for file size,  yes, the sqlite dbs are quite large, but we've already been warning people about WOF data size forever. It should be ok.